src/zepben/eas/client/work_package.py

```diff
@@ -127,6 +129,78 @@ class MeterPlacementConfig:
     """The ID of the meter group to use for populating EnergyMeters at EnergyConsumers."""
 
 
+@dataclass
+class FixedTimeLoadOverride:
+
```

Comment:
Remove the override suffix from all these properties. It is a bit redundant given that the class itself already tells us they are an override.